rust/src/smb/smb2.rs

```diff
@@ -297,7 +297,7 @@ pub fn smb2_write_request_record<'b>(state: &mut SMBState, r: &Smb2Record<'b>)
             };
 
             let mut set_event_fileoverlap = false;
-            let found = match state.get_file_tx_by_fuid(&file_guid, Direction::ToServer) {
```

Comment:
Do we still need simple function `get_file_tx_by_fuid` ?

Reply:
Yes it's used by the close command and one other case.